client/store/orders/selectors.js

```diff
@@ -1,7 +1,49 @@
 /** @format */
 
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { getJsonString } from 'store/utils';
+import { ERROR } from 'store/constants';
+
+/**
+ * Returns orders for a specific query.
+ *
+ * @param  {Object} state     Current state
+ * @param  {Object} query     Report query paremters
+ * @return {Object}           Report details
+ */
+function getOrders( state, query = {} ) {
+	return get( state, [ 'orders', getJsonString( query ) ], [] );
```

Comment:
Should this return an empty object by default, or should the JSDOC @return be `{Array}`?